.eslintrc.js

```diff
@@ -18,10 +17,14 @@ module.exports = {
   },
   plugins: ['react'],
   rules: {
-    'react/jsx-filename-extension': [
-      2,
+    'no-shadow': 0,
+    'object-curly-newline': [
+      'error',
       {
-        extensions: ['.js'],
+        ObjectExpression: { multiline: true, consistent: true },
+        ObjectPattern: { multiline: true, consistent: true },
+        ImportDeclaration: { multiline: true, consistent: true },
+        ExportDeclaration: { multiline: true, consistent: true },
       },
     ],
     'react/no-array-index-key': 0,
```

Comment:
Since we've decided not to use `index` as item key in arrays, I suppose we could set it to throw warnings?

Reply:
But how?

Reply:
Huh, I thought we could achieve that by putting `1` instead of `0`, but I see that you didn't do that for a reason :)